### PR TITLE
separate out existing tools

### PR DESCRIPTION
**Pull request by hamishun**

### CODE REVIEW

**Comment by hamishun** [comment]
A few integration tests I missed to separate, so don't merge, yet.

**Comment by hamishun** [comment]
OK separation done.

File: kit/tools/README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # HE Tools
 
-The File `healg.py` contains useful commands to finds HE parameters based on user constraints. Next sections describe the details about them.
+The tools commands for `hekit` contain useful commands to finds HE parameters
```
Comment (jlhcrawford) [inline kit/tools/README.md:3]:
```suggestion
The tools for `hekit` consist of useful commands for finding HE parameters
```

File: kit/tools/README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+based on user constraints. Next sections describe the details about them.
```
Comment (jlhcrawford) [inline kit/tools/README.md:4]:
```suggestion
based on user constraints. The following sections describe their details and usage.
```

File: kit/tools/README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
 ## Dependencies
 - python >= 3.8
 
 ## gen-primes
 
-The command `hekit gen-primes` generates a list of sorted primes in range [n, m] where n and m are positive integers
+The command `hekit gen-primes` generates a list of sorted primes in range [n,
+m] where n and m are positive integers
```
Comment (jlhcrawford) [inline kit/tools/README.md:12]:
```suggestion
m] where n and m are positive integers.
```

**Review by jlhcrawford — CHANGES_REQUESTED** [review]

**Review by denCalderon — COMMENTED** [review]
minor changes

File: tests/test_integration_algebras.py [hunk 4]
```diff
@@ -190,8 +118,8 @@ def test_healg_negative_arg_d(hekit_path):
 
 
 @pytest.mark.skip(reason="Not realistic test because SW cannot handle that amount in d")
-def test_healg_max_arg_d(hekit_path):
-    """Verify that gen-primes cmd is excuted correctly when
+def test_max_arg_d(hekit_path):
+    """Verify that gen-primes cmd is executed correctly when
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:122]:
```suggestion
    """Verify that the algebras cmd is executed correctly when
```

File: tests/test_integration_algebras.py [hunk 5]
```diff
@@ -190,8 +118,8 @@ def test_healg_negative_arg_d(hekit_path):
...
     d is equal to sys.maxsize"""
     cmd = f"{hekit_path} -p 7 -d {sys.maxsize} --no-header"
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:124]:
```suggestion
    cmd = f"{hekit_path} algebras -p 7 -d {sys.maxsize} --no-header"
```

File: tests/test_integration_algebras.py [hunk 6]
```diff
@@ -177,7 +105,7 @@ def test_healg_arg_d(hekit_path):
 
 
 @pytest.mark.xdist_group(name="arg_group")
-def test_healg_negative_arg_d(hekit_path):
+def test_negative_arg_d(hekit_path):
     """Verify that gen-primes cmd triggers an error when
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:109]:
```suggestion
    """Verify that the algebras cmd triggers an error when
```

File: tests/test_integration_algebras.py [hunk 7]
```diff
@@ -159,8 +87,8 @@ def test_healg_max_arg_p(hekit_path):
 
 
 @pytest.mark.xdist_group(name="arg_group")
-def test_healg_arg_d(hekit_path):
-    """Verify that gen-primes cmd is excuted correctly when
+def test_arg_d(hekit_path):
+    """Verify that gen-primes cmd is executed correctly when
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:91]:
```suggestion
    """Verify that the algebras cmd is executed correctly when
```

File: tests/test_integration_algebras.py [hunk 8]
```diff
@@ -146,8 +74,8 @@ def test_healg_negative_arg_p(hekit_path):
 
 
 @pytest.mark.xdist_group(name="arg_group")
-def test_healg_max_arg_p(hekit_path):
-    """Verify that gen-primes cmd is excuted correctly when
+def test_max_arg_p(hekit_path):
+    """Verify that gen-primes cmd is executed correctly when
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:78]:
```suggestion
    """Verify that the algebras cmd is executed correctly when
```

File: tests/test_integration_algebras.py [hunk 9]
```diff
@@ -133,7 +61,7 @@ def test_healg_arg_p(hekit_path):
 
 
 @pytest.mark.xdist_group(name="arg_group")
-def test_healg_negative_arg_p(hekit_path):
+def test_negative_arg_p(hekit_path):
     """Verify that gen-primes cmd triggers an error when
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:65]:
```suggestion
    """Verify that the algebras cmd triggers an error when
```

File: tests/test_integration_algebras.py [hunk 10]
```diff
@@ -115,8 +43,8 @@ def test_healg_arg_no_header(hekit_path):
 
 
 @pytest.mark.xdist_group(name="arg_group")
-def test_healg_arg_p(hekit_path):
-    """Verify that gen-primes cmd is excuted correctly when
+def test_arg_p(hekit_path):
+    """Verify that gen-primes cmd is executed correctly when
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:47]:
```suggestion
    """Verify that the algebras cmd is executed correctly when
```

File: tests/test_integration_algebras.py [hunk 11]
```diff
@@ -97,8 +25,8 @@ def test_healg_arg_header(hekit_path):
 
 
 @pytest.mark.xdist_group(name="arg_group")
-def test_healg_arg_no_header(hekit_path):
-    """Verify that algebras cmd is excuted correctly when
+def test_arg_no_header(hekit_path):
+    """Verify that algebras cmd is executed correctly when
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:29]:
```suggestion
    """Verify that the algebras cmd is executed correctly when
```

File: tests/test_integration_algebras.py [hunk 12]
```diff
@@ -6,81 +6,9 @@
...
-        in act_result.stderr
-    )
-    assert 0 != act_result.returncode
-
-
 @pytest.mark.xdist_group(name="arg_group")
-def test_healg_arg_header(hekit_path):
-    """Verify that gen-primes cmd is excuted correctly
+def test_arg_header(hekit_path):
+    """Verify that gen-primes cmd is executed correctly
```
Comment (jlhcrawford) [inline tests/test_integration_algebras.py:11]:
```suggestion
    """Verify that the algebras cmd is executed correctly
```

**Review by jlhcrawford — CHANGES_REQUESTED** [review]

**Pull request merged**